integration_test/README.md

```diff
@@ -164,7 +162,7 @@ data directory and the test runner before it is really meeting our needs.
 ### Adding a new third-party application
 
 You will need to add and modify a few files. Start by adding your new
-application to `agent/ops-agent/<linux_or_windows>/supported_applications.txt`
```

Comment:
same here, this change should have been part of https://github.com/GoogleCloudPlatform/ops-agent/pull/484.